README.md

```diff
@@ -1 +1,128 @@
-# grid3_client_go
\ No newline at end of file
+# **grid3_go:**
+
+grid3_go is a go client created to interact with threefold grid. It should manage CRUD operations for deployments on the grid.
+
+- ## **Grid3_go flow:**
+
+  1. the deployment manager will be initialized with identity/network information.
+  2. deployment manager will expose the `deploy` which the user will call it and give it old/new deployments
+  3. `deploy` method will do the following
+     - internally call `calculateChanges` method which
+       1. loads old deployments using their ids from teh grid
+       2. determine which things needs to be created, updated and deleted
```

Comment:
does things here mean workloads?

Reply:
yes, the deployment include workloads inside it